### PR TITLE
Add blog post about improved filters for notifications

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

**Review by saraycp — CHANGES_REQUESTED** [review]

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Notifications are back, and they come with the strength of a pretty and revamped filtering page.
+Let's get into the details, and keep reading.
+We have prepared more improvements on how you get notified by OBS.
+
+# Improved Filters
+
+If you want to be informed about what happens in your OBS instance, you most likely deal with lots of notifications.
+We have made it easier for you to focus on what you need by enhancing the filtering of your notifications page.
+Among the improvements on performance, you can now:
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md]: Applied. Thanks!

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- perform a "click and submit" on any option
+- benefit from a more grouped view of projects and groups filters
+- filter your projects and groups with a text box
+
+Just take a look at this example:
+
+![](/images/posts/2024-06-12/notifications-filters.gif)
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:23]: True. I'll create another Gif including that part.

[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:23]: Done. Please take a look again.

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Improved Filtering For Your Notifications
+category: development
+---
+
+Notifications are back, and they come with the strength of a pretty and revamped filtering page.
+Let's get into the details, and keep reading.
+We have prepared more improvements on how you get notified by OBS.
```
Comment (danidoni) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:9]: I would use one of those lines, but not both.

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+# Improved Filters
+
+If you want to be informed about what happens in your OBS instance, you most likely deal with lots of notifications.
```
Comment (danidoni) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:13]: I think this is not about what happens in the instance. Most users don't care about the instance, but the projects, packages and such things they work on. The notifications are for the user to stay on top of their work.

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+We have made it easier for you to focus on what you need by enhancing the filtering of your notifications page.
+In addition to the performance improvements, you can now:
+
+- perform a "click and submit" on any option
```
Comment (danidoni) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:17]: I would rephrase this as something in the line of "less clicking" the details are not important

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 3 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:9]: Yes, done.

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 4 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:13]: I rephrased the sentence. Please take a look.

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 5 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:17]: I also reprhrase this one.

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+Notifications are back, and they come with the strength of a pretty and revamped filtering page.
+Let's get into the details, and keep reading.
+
+# Improved Filters
+
+If you want to stay on top of your work in your OBS instance, you most likely deal with lots of notifications.
+We have made it easier for you to focus on what you need by enhancing the filtering of your notifications page.
+Basically, you can now:
+
+- click and expect new results, meaning that your clicks wil be reduced
```
Comment (saraycp) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:16]: @suggestion
```
- click and expect new results, meaning that your clicks will be reduced
```

But I'd suggest something like `get results as soon as you click on any filter` @danidoni what do you think?

**Review by saraycp — CHANGES_REQUESTED** [review]

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 6 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:16]: I like it. Changed.

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Improved Filtering For Your Notifications
+category: development
+---
+
+Notifications are back, and they come with the strength of a pretty and revamped filtering page.
```
Comment (hennevogel) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:7]: > Notifications are back

Where have they been? :)

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+Let's get into the details, and keep reading.
```
Comment (hennevogel) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:8]: Might be some cultural thing but if some blog post tells me to keep reading I find that pretty arrogant...

Comment (hennevogel) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:8]: Might be some cultural thing but if some blog post tells me to keep reading I find that pretty arrogant...

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 7 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:7]: I see. I changed the sentence.

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 8 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:8]: I changed that sentence too. Please take a look again.

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 4 above)
Comment (danidoni) [reply _posts/development/2024-06-12-notifications-improved-filtering.md]: Better!

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 5 above)
Comment (danidoni) [reply _posts/development/2024-06-12-notifications-improved-filtering.md]: Perfect

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Improved Filtering For Your Notifications
+category: development
+---
+
+Notifications have been always part of your workflow.
+Now we are glad to present a set of changes including a pretty and revamped filtering page.
+Let's get into the details and enjoy the reading!
```
Comment (danidoni) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:9]:
```suggestion
Let's get into the details!
```
What about that?

Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md:9]: I like it too. :+1:

File: _posts/development/2024-06-12-notifications-improved-filtering.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Improved Filtering For Your Notifications
+category: development
+---
+
+Notifications have always been part of your workflow.
+Now, we are glad to present a set of changes, including a pretty and revamped filtering page.
```
Comment (saraycp) [inline _posts/development/2024-06-12-notifications-improved-filtering.md:8]:
```suggestion
Now, we are glad to present a set of changes, including a revamped filtering section.
```

Sorry, I didn't realize before. We don't have a "filtering page", that's why I suggest "filtering section".
And if it's "pretty" or not depends on users taste. It's something we shouldn't say.

**Review by saraycp — CHANGES_REQUESTED** [review]

File: _posts/development/2024-06-12-notifications-improved-filtering.md (hunk 10 above)
Comment (eduardoj) [reply _posts/development/2024-06-12-notifications-improved-filtering.md]: Applied. Thanks.

**Review by saraycp — APPROVED** [review]

**Review by danidoni — APPROVED** [review]
